test/lib/ansible_test/_util/controller/sanity/pylint/plugins/deprecated.py

```diff
@@ -172,7 +172,7 @@ def _check_date(self, node, date):
 
     def _check_version(self, node, version, collection_name):
         if not isinstance(version, (str, float)):
-            self.add_message('invalid-version', node=node, args=(version,))
+            self.add_message('ansible-invalid-deprecated-version', node=node, args=(version,))
```

Comment:
Shouldn't this check `collection_name` as is done below and then return either `ansible-invalid-deprecated-version` or `collection-invalid-deprecated-version`?

Reply:
Oops, this has been addressed.